R/gh_response.R

```diff
@@ -21,7 +21,7 @@ gh_process_response <- function(response) {
       headers = heads,
       message = paste0("GitHub API error (", status_code(response), "): ",
                        heads$`status`, "\n  ", res$message, "\n")
-    ), class = c("condition", "error"))
+    ), class = c("github_error", status_code(response), "condition", "error"))
```

Comment:
Also, can you please switch `condition` and `error`, `error` is more specific and should come first. (I know you didn't do that, but we might as well fix it, too.)